.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    env: {
+        es2021: true,
+        node: true
+    },
+    root: true,
+    extends: ["prettier"],
+    parser: "@typescript-eslint/parser",
+    plugins: ["@typescript-eslint"],
+    parserOptions: {
+        ecmaVersion: "latest",
+        sourceType: "module",
+        project: true,
+        tsconfigRootDir: __dirname
+    },
+    rules: {
```

Comment:
I added this to capture some of the conventions that you raised - I think conventions are a vital part of collaborative work, so hopefully this will help to keep people on the same page